build.sbt

```diff
@@ -27,13 +27,16 @@ inThisBuild(
 addCommandAlias("fmt", "all scalafmtSbt scalafmt test:scalafmt")
 addCommandAlias("check", "all scalafmtSbtCheck scalafmtCheck test:scalafmtCheck")
 
+addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")
```

Comment:
I would not recommend using this in ZIO projects. It will enable you to write code that does not work on Dotty. It's good for user-land apps, though.

Reply:
sure I will remove it